mixer/adapter/signalfx/config/config.proto

```diff
@@ -75,4 +126,17 @@ message Params {
     // This will be rounded to the nearest second and rounded values less than
     // one second are not valid. Defaults to 10 seconds if not specified.
     google.protobuf.Duration datapoint_interval = 4 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
+
+    // Optional.  If set to true, metrics won't be sent (but trace spans will
+    // be unless otherwise disabled).
+    bool disable_metrics = 5;
+
+    // Optional.  If set to true, trace spans won't be sent (but metrics will
+    // be unless otherwise disabled).
+    bool disable_tracing = 6;
+
+    // Optional.  The number of set of trace spans that the adapter will buffer
+    // before dropping them.  This defaults to 1000 spans but can be configured
+    // higher if needed.  An error message will be logged if spans are dropped.
+    uint32 tracing_buffer_size = 7;
```

Comment:
Do you want to consider a sampling model as well? Sending all traces can be prohibitive in a high traffic situation.

Reply:
Yes, I will add that.

Is the plan that the telemetry server will receive and pass on all spans and delegate sampling choices entirely to the adapters?

Reply:
@douglas-reid What's the end to end story here?

Reply:
For Mixer-driven tracing?  Right now, sampling is controlled by the adapter (as long as tracing is 100% enabled in the mesh -- which it is by default). Moving forward, I expect there to mesh-level sampling controls via a full API that can handle sampling configuration.